dbrp/service.go

```diff
@@ -0,0 +1,489 @@
+package dbrp
+
+// The DBRP Mapping `Service` maps database, retention policy pairs to buckets.
```

Comment:
@aanthony1243 here are the comments on doing relational stuff with a kv backend 👍